src/services/ledger/index.ts

```diff
@@ -179,6 +184,16 @@ export const LedgerService = (): ILedgerService => {
       const { balance } = await MainBook.balance({
         account: liabilitiesWalletId,
       })
+      if (balance < 0) {
+        addAttributesToCurrentSpan({
+          "getWalletBalance.error.invalidBalance": `${balance}`,
+          "error": true,
+          "error.level": ErrorLevel.Warn,
+        })
+        addEventToCurrentSpan("exception", {
```

Comment:
Have you considered how to query this? Isn't `exception` usually associated with an error and recorded as `span.recordException`?

Reply:
Addressed here: https://github.com/GaloyMoney/galoy/pull/1164/commits/1a84907f32ea963633b3d1caff6ed6b1fbba71a6

Do we want exceptions to only be recorded when an entire use-case returns an error and terminates (current behaviour), or is it ok to have exceptions recorded mid-process like this?